storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java

```diff
@@ -192,16 +192,14 @@ public DefaultStorIOSQLite build() {
         private final Map<Class<?>, SQLiteTypeMapping<?>> indirectTypesMappingCache
                 = new ConcurrentHashMap<Class<?>, SQLiteTypeMapping<?>>();
 
-        /**
-         * Guarded by {@link #lock}.
-         */
-        private int numberOfRunningTransactions = 0;
+        @NonNull
+        private AtomicInteger numberOfRunningTransactions = new AtomicInteger(0);
 
         /**
          * Guarded by {@link #lock}.
          */
         @NonNull
-        private final Set<Changes> pendingChanges = new HashSet<Changes>();
+        private Set<Changes> pendingChanges = new HashSet<Changes>(5);
```

Comment:
It's just randomly selected number used to avoid unnecessary memory allocation, I guess `5` is not big and not small for this, so, why not :)